src/components/mes-signature-pad/mes-signature-pad.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, ViewChild, EventEmitter, Output } from '@angular/core';
+import { SignaturePad } from 'angular2-signaturepad/signature-pad';
+
+@Component({
+  selector: 'mes-signature-pad',
+  templateUrl: 'mes-signature-pad.html',
+})
+export class MesSignaturePadComponent {
+  @Output() changedDataEvent = new EventEmitter<string>();
```

Comment:
We shouldn't be using event emitters , this should be done via the store

Reply:
trying to keep the store away from the component, which may at some point not have a store to work with if it's used somewhere else.